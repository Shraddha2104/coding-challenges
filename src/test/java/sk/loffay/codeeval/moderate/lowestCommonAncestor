20 10
29 20
52 29
30 8
10 29
3 52
20 52
8 20
3 52
3 8
3 29
3 52
30 29
8 52
29 52
8 52
20 29
8 20
30 8
52 30
