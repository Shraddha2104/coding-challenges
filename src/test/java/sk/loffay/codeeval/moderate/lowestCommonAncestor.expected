20
20
30
30
20
30
30
8
30
8
8
30
30
30
30
30
20
8
30
30
